tarofreighter/chain_porter.go

```diff
@@ -418,28 +420,66 @@ func (p *ChainPorter) storeAndTransferProofs(pkg *sendPackage) {
 	log.Debugf("Updated proofs for sender and receiver (new_len=%d)",
 		senderProof.NumProofs())
 
+	pkg.SendState = SendStateReceiverProofTransfer
```

Comment:
For this to work properly on restart, then this also needs to be committed on disk (the state of the send itself).

Reply:
As discussed, will leave as future work. Tracking issue: https://github.com/lightninglabs/taro/issues/257